sdk/cognitivelanguage/ai-language-text-authoring/.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Is this file needed?

Reply:
No, I believe we don't need this file for our purposes